mu4e-thread-folding.el

```diff
@@ -150,8 +151,8 @@ This uses the mu4e private API and this might break in future releases."
         (let ((overlay-priority     -60) 
               (folded               (if (string= mu4e-thread-folding-default-view 'folded) t nil))
               
-              (child-overlay        nil)
-              (child-prefix-overlay nil)
+              (_child-overlay        nil)
```

Comment:
Why the `_`?

Reply:
These are lexical variables which aren't used at all, so the byte-compiler complains.

The `_` marks them as *intentionally* unused, which suppresses the warning.

Reply:
You could alternatively not bind these at all; but I figured that maybe you were planning to use them later.

Reply:
I see. the _child can be removed I think. For the root, I'll need to enforce initial folding (which is buggy right now)